FHEM/14_SD_UT.pm

```diff
@@ -560,10 +560,14 @@ sub SD_UT_Define($$) {
 	if (($a[2] eq "SA_434_1_mini" || $a[2] eq "QUIGG_DMV") && not $a[3] =~ /^[0-9a-fA-F]{3}/s) {
 		return "wrong HEX-Value! ($a[3]) $a[2] HEX-Value to short | long or not HEX (0-9 | a-f | A-F){3}";
 	}
-	### [4] checks Neff SF01_01319004 & BOSCH SF01_01319004_Typ2 & Chilitec_22640 & Manax & ESTO KL_RF01###
-	if (($a[2] eq "SF01_01319004" || $a[2] eq "SF01_01319004_Typ2" || $a[2] eq "Chilitec_22640" || $a[2] eq "Manax" || $a[2] eq "KL_RF01") && not $a[3] =~ /^[0-9a-fA-F]{4}/s) {
+	### [4] checks Neff SF01_01319004 & BOSCH SF01_01319004_Typ2 & Chilitec_22640 & ESTO KL_RF01###
+	if (($a[2] eq "SF01_01319004" || $a[2] eq "SF01_01319004_Typ2" || $a[2] eq "Chilitec_22640" || $a[2] eq "KL_RF01") && not $a[3] =~ /^[0-9a-fA-F]{4}/s) {
 		return "wrong HEX-Value! ($a[3]) $a[2] HEX-Value to short | long or not HEX (0-9 | a-f | A-F){4}";
 	}
+	### [6] checks Manax | mumbi ###
+	if ($a[2] eq "RC_10" && not $a[3] =~ /^[0-9a-fA-F]{4}_([ABCD]|[a][l][l])$/s) {
```

Comment:
rein Interessehalber
`not $a[3] =~ /^[0-9a-fA-F]{4}_([ABCD]|[a][l][l])$/s`

Die Regex soll prüfen, ob diese Zeichenfolge nicht vorkommt oder? Dafür gibt es ` $a[3] !~`.


Den leztzten Teil kannst Du etwas vereinfacht schreiben, wenn ich richtig verstanden habe das $a[3] den Inhalt AAAA_a11 haben könnte:
`^[0-9a-fA-F]{4}_(?:[ABCD]|a11)$`

Reply:
So sollte nun die Richtige Fassung sein ;-)

Reply:
Deine Fassung war nicht falsch. Nur etwas schwierig für mich zu lesen